tests/run/t3719.scala

```diff
@@ -1,4 +1,4 @@
-// scalajs: --skip --pending
+// scalajs: --skip
```

Comment:
This test uses `extends Val` without explicit name, which is not supported (and was never supported in Scala 2 either).